sched/generators.go

```diff
@@ -50,3 +52,57 @@ func GenJobId() gopter.Gen {
 		return gopter.NewGenResult(string(result), gopter.NoShrinker)
 	}
 }
+
+func makeDomainJobFromThriftJob(thriftJob *schedthrift.Job) *Job {
+
+	var domainJob = Job{}
+	domainJob.Id = thriftJob.ID
+	sagaLogJobDef := thriftJob.JobDefinition
+
+	var schedJobDef = JobDefinition{}
+	schedJobDef.JobType = sagaLogJobDef.JobType
+	schedJobDef.Tasks = make(map[string]TaskDefinition)
+	for taskName, sagaLogTaskDefTask := range sagaLogJobDef.Tasks {
+		var argvs []string
+		argvs = sagaLogTaskDefTask.Command.Argv
+		var envVars map[string]string
+		envVars = sagaLogTaskDefTask.Command.EnvVars
+		timeout := time.Duration(sagaLogTaskDefTask.Command.Timeout)
+		snapshotId := sagaLogTaskDefTask.Command.SnapshotId
+		command := runner.NewCommand(argvs, envVars, timeout, snapshotId)
```

Comment:
You create all of these varaibles just to use them once...

Could you just do:
  cmd := task.Command
 runner.NewCommand(cmd.Argv, cmd.EnvVars, cmd.Timeout, cmd.SnapshotId)

?